llvm/test/CodeGen/AMDGPU/dagcombine-select-frem.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: llc -mtriple=amdgcn -mcpu=gfx900 -verify-machineinstrs < %s | FileCheck -enable-var-scope -check-prefix=GCN %s
+; XFAIL: *
```

Comment:
No xfailed test, especially one with checks

Reply:
This was extracted from dagcombine-select.ll. I deleted this file and added moved the TODO to the original test case in the former file.